src/openrct2/actions/FootpathPlaceAction.hpp

```diff
@@ -88,7 +88,7 @@ DEFINE_GAME_ACTION(FootpathPlaceAction, GAME_COMMAND_PLACE_PATH, GameActionResul
             return MakeResult(GA_ERROR::DISALLOWED, STR_CANT_BUILD_FOOTPATH_HERE, STR_TOO_HIGH);
         }
 
-        if (_direction != 0xFF && _direction > 15)
+        if (_direction != INVALID_DIRECTION && !direction_valid(_direction))
```

Comment:
This is a change in behaviour - direction_valid() restricts to 0-3 - but as discussed on Gitter, the new behaviour is more correct.